lib/features/popup-menu/ColorMenuProvider.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a commercial license.
+ * You may not use this file except in compliance with the commercial license.
+ */
+
+var COLORS = [{
```

Comment:
We should align closely here with [bpmn-js-color-picker](https://github.com/bpmn-io/bpmn-js-color-picker/blob/master/colors/ColorPopupProvider.js#L20), including but not limited to 

* Do not copy Camunda modeler code (proprietary license)
* Clear colors when the user sets "white"
* Potential other topics (i'm not aware of any to date)

Reply:
> Do not copy Camunda modeler code (proprietary license header)

Oops, I fixed thats.

> Clear colors when the user sets "white"

Also addressed this.

I also took a look at the bpmn-js-color-picker, and didn't notice any other discrepancies

Reply:
:+1: